tools/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.vendoring]
+destination = "third_party/"
+requirements = "requirements_vendor.txt"
+namespace = ""
+patches-dir = "third_party_patches"
+protected-files = [
+  "README.md",
+]
+
+[tool.vendoring.license.fallback-urls]
+atomicwrites = "https://raw.githubusercontent.com/untitaker/python-atomicwrites/refs/tags/1.1.5/LICENSE"
+more_itertools = "https://raw.githubusercontent.com/more-itertools/more-itertools/refs/tags/4.2.0/LICENSE"
+pathlib2 = "https://raw.githubusercontent.com/jazzband/pathlib2/refs/tags/2.3.5/LICENSE.rst"
+webencodings = "https://raw.githubusercontent.com/gsnedders/python-webencodings/refs/tags/v0.5.1/LICENSE"
+
+[tool.vendoring.transformations]
+drop = [
+  "*.dist-info",
```

Comment:
Would not dropping these help with the pytest-asyncio situation?

Reply:
I'll need to check, but I think this was mostly motivated by changes between pre-release and stable vendoring, and getting consistent output between the two.